admin/init.sls

```diff
@@ -22,22 +22,37 @@ UTC:
     - mode: 644
     - source: salt://{{ tpldir }}/files/hosts
 
-sshkeys-dir:
-  file.directory:
-    - name: {{ root.home }}/.ssh
-    - user: {{ root.user }}
-    - group: {{ root.group }}
-    - mode: 700
-
-sshkeys:
+/etc/sshd_config:
```

Comment:
We should keep the states that set up the SSH keys for the root user, as we don't want to lock ourselves out (and at least for now want to keep installing root SSH keys on new machines).

Reply:
Fixed :)